### PR TITLE
Update signalr.md

### PR DESCRIPTION
**Pull request by GeeSuth**
remove the duplicate line.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/tutorials/signalr.md](https://github.com/dotnet/AspNetCore.Docs/blob/57537f6b8a55439d3c82638607a06f5c194afabf/aspnetcore/tutorials/signalr.md) | [Customer intent: As a developer, I want to get a quick proof-of-concept app running, so I can get a practical introduction to ASP.NET Core SignalR.](https://review.learn.microsoft.com/en-us/aspnet/core/tutorials/signalr?branch=pr-en-us-30940) |

### CODE REVIEW

File: aspnetcore/tutorials/signalr.md [hunk 1]
```diff
@@ -155,7 +155,6 @@ The output looks like the following example:
 
 ```console
 wwwroot/js/signalr/dist/browser/signalr.js written to disk
```
Comment (GeeSuth) [inline aspnetcore/tutorials/signalr.md:157]: And maybe will be better to add the exactly what you got when run the command 
```
Downloading file https://unpkg.com/@microsoft/signalr@latest/dist/browser/signalr.js...
wwwroot/js/signalr/dist/browser/signalr.js written to disk
Installed library "@microsoft/signalr@latest" to "wwwroot/js/signalr"
```

Comment (guardrex) [reply aspnetcore/tutorials/signalr.md:157]: Hello @GeeSuth ... Is that complete? Does it actually have an ellipsis ("`...`") at the end of the "Downloading" line, or did you not show the full line?

Comment (guardrex) [reply aspnetcore/tutorials/signalr.md:157]: I'll go ahead with an ellipsis there. I think you mean that the line is quite a bit longer.

**Review by guardrex — APPROVED** [review]
Thanks @GeeSuth! 🚀

File: aspnetcore/tutorials/signalr.md (hunk 1 above)
Comment (GeeSuth) [reply aspnetcore/tutorials/signalr.md:158]: @guardrex sorry for delay, it's actually ended with ellipsis 

image


Thank you for merge!

**Pull request merged**